src/onemod/utils/data_io_handler.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+
+from polars import DataFrame, read_parquet
+
+
+class DataIOHandler:
```

Comment:
What are your thoughts about writing our own handler vs. using `pplkit.DataInterface`? Besides read/write, `DataInterface` also keeps track of directory structure, but we could create a OneMod version of that too.